src/shared/cgroup-setup.c

```diff
@@ -17,6 +17,67 @@
 #include "fileio.h"
 #include "user-util.h"
 #include "fd-util.h"
+#include "virt.h"
+
+static bool cg_is_unified_possible(void) {
```

Comment:
maybe rename this cg_any_controller_used_for_v1()

Reply:
ack